securedrop_client/resources/css/sdclient.css

```diff
@@ -16,6 +16,14 @@ QWidget {
     color: #fff;
 }
 
+#SyncStatusBar {
+    font-family: 'Source Sans Pro';
+    font-weight: 600;
+    font-size: 12px;
+    color: #d3d8ea;
+    background: transparent;
```

Comment:
When testing locally (on a mac), without `background: transparent;` this element and `#ActivityStatusBar` both appear grey. 

<img width="947" alt="Screenshot 2022-12-15 at 14 28 35" src="https://user-images.githubusercontent.com/17057932/207886437-658b5bc6-8d85-4294-bd18-befaa187546b.png">

Reply:
That's probably inherited from the `QStatusBar` which is an element of chrome (typically displayed at the bottom of the window). What "chome" looks like varies across platforms. I think that adapting the background to our use (even development use in this case) is the right thing to do.

Reply:
Any reason why you didn't also set the transparent background for `#ActivityStatusBar` @zekehuntergreen ?

Reply:
Good call - I've made that change